spec/controllers/idv/doc_auth_controller_spec.rb

```diff
@@ -235,6 +235,16 @@
         attention_with_barcode: false,
       }
     end
+    let(:hard_fail_result) do
```

Comment:
what is a hard fail?

Reply:
A hard fail refers to cases where we should not provide the user with the IPP option, e.g. in cases of suspected fraud.

Reply:
got it